docs/app/pages/components/components-sections/timeline/timeline/timeline.component.html

```diff
@@ -0,0 +1,38 @@
+<div class="row">
+    <div class="col-md-10">
+        <button class="btn button-primary" (click)="addEvent()">Add New Event</button>
+        <ux-timeline>
+            <ux-timeline-event *ngFor="let ticket of tickets" [badgeColor]="ticket.color" [badgeTitle]="ticket.shortDate"
+                [title]="ticket.title" [timestamp]="ticket.date">
+        
+                <p>Ticket
+                    <a [href]="ticket.url">{{ticket.id}}</a>
+                    was {{ticket.status}} {{ticket.assignee}}.
```

Comment:
Probably want a "by" in here or something.

Reply:
I moved the preposition into the 'status' property to allow use of phrases like 'assigned for development to' as well as 'recorded by' and 'reviewed by'. Is this OK?

Reply:
Prefer not. One does not store sentences in a database column - apart from anything else, it makes localization a pain. As stated before, displays which require more detail would use an alternate template. But let's keep it simple. (Maybe call it action instead of status and the objects events instead of tickets, if that sits better with you.)

````
interface TicketEvent {
  //...
  action: 'opened' | 'closed' | 'updated';
}
````

````
<p>Ticket
    <a [href]="event.ticketUrl">{{event.ticketId}} - {{event.ticketTitle}}</a>
    was {{event.action}} by {{event.user}}.
</p>
````

Seem okay to you?